installer/installer.py

```diff
@@ -522,6 +524,98 @@ def cmd_install_netmodules(public_slave=False):
     return
 
 
+def cmd_install_cni(public_slave=False):
+    """
+    Install Calico's CNI plugin.  A successful completion of the task
+    indicates successful installation
+
+    :param public_slave: Flag indicating if this is a public slave.
+    """
+    # Load the current Calico install info for Docker, and the current Docker
+    # daemon configuration.
+    install_config = load_config(CNI_INSTALL_CONFIG)
+
+    # Before starting the install, check that we are able to locate the agent
+    # process - if not there is not much we can do here.
+    if not install_config:
+        _log.debug("Have not started installation yet")
```

Comment:
This log messaged is just used to track the execution flow when debugging, so I think it qualifies as a debug message